### PR TITLE
added gsod section

### LINKED ISSUE #706: [NG] Migrate site data for the GSoD page to the layer5-ng site
**Description**
Currently, the internship page dedicated to [GSoD](https://layer5ng.netlify.app/careers/programs/gsod/2020) on the [layer5-ng site](https://layer5ng.netlify.app/) points to a 404 error. 

**Expected Behavior**

The data from the [GSoD page](https://layer5.io/programs/gsod/2020) on the [layer5.io site](https://layer5.io/) needs to be migrated to the layer5-ng site. 

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by codetheorem**
**Description**

Adds the Gsod 2020 page to layer5-ng.

This PR fixes #706 

**Notes for Reviewers**
I have tested my code by creating a temporary page which I removed later as there is no such page currently available in which changes can viewed.

**Screen Shots**

screencapture-localhost-8000-career-1600195222929

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Nikhil-Ladha** [comment]
Add a temp link in the footer for this page. It can be removed later on.

File: src/sections/Gsod/gsod.style.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+    -webkit-tap-highlight-color: transparent;
+    vertical-align: middle;
+    z-index: 1;
+    -webkit-transition: 0.3s ease-out;
+    transition: 0.3s ease-out;
+}
+.btn, .btn-large {
```
Comment (leecalcote) [inline src/sections/Gsod/gsod.style.js:29]: @Tanuj22 what is our strategy with respect to the reuse of global styles?

Comment (Tanuj22) [reply src/sections/Gsod/gsod.style.js:29]: These styles won't be needed. We are going to use a separate button component already made. (Was just waiting for @codetheorem to app a temp link for doing a review)

**Comment by codetheorem** [comment]
@Tanuj22  Please review the changes.

File: src/sections/Gsod/index.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { Container, Row, Col } from "../../reusecore/Layout";
+import logo from "../../images/layer5/gsod.png";
+import GSODSectionWrapper from "./gsod.style";
+
+const Gsod = () => {
+    return (
+        <GSODSectionWrapper>
+            <Container>
+                <div style={{zIndex: "20"}}>
```
Comment (Tanuj22) [inline src/sections/Gsod/index.js:10]: Remove inline styles.

**Review by Tanuj22 — CHANGES_REQUESTED** [review]
The video is getting cut from left side 
Screenshot from 2020-09-16 19-06-46

**Comment by codetheorem** [comment]
@Tanuj22 @Nikhil-Ladha Please review the changes.

**Comment by Nikhil-Ladha** [comment]
The secondary heading `GSoD and Layer5` can be made the primary heading and the second one can be removed. Also, add a section similar to the gsoc page like what is Layer5 and there add the video. Take a reference to the gsoc page and update the page accordingly.

**Comment by Nikhil-Ladha** [comment]
@leecalcote shall we keep the `Participate` button, as we don't have the program active anymore, it has expired.

**Comment by codetheorem** [comment]
Updated @Nikhil-Ladha @Tanuj22

**Comment by codetheorem** [comment]
No participate button is there on the page.

**Comment by Tanuj22** [comment]
@codetheorem Great work. Can you run `npm run lint` to fix the indentations? Then we are good to go!

**Comment by codetheorem** [comment]
@Tanuj22 done all the changes.

**Comment by codetheorem** [comment]
Thank you for your guidance @Tanuj22 @Nikhil-Ladha . Bear me for some more PRs. 😁

**Comment by Tanuj22** [comment]
> Thank you for your guidance @Tanuj22 @Nikhil-Ladha . Bear me for some more PRs.

Keep em coming :muscle:

**Comment by leecalcote** [comment]
Who-hoo!

**Pull request merged**